spec/inheritance_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+RSpec.describe "inheritance" do # rubocop:disable RSpec/DescribeClass
```

Comment:
I don't feel too strongly but I have a preference for keeping these tests in the `spec/memo_wise_spec.rb` file because we have a whole host of other inheritance examples in there and I find it easier to mentally sort out what cases we've tested (for example, this isn't just "simple inheritance," it's also class methods defined with `self.` as opposed to the other types of methods we support).

Thoughts?

Reply:
I think it's more clear in a separate file (and could even migrate the existing ones here). The existing ones are inserted as parts of other contexts instead of all in one place, and I think it's more straightforward to keep inheritance as its own set of specs. Let me know your thoughts